### DISCUSSION TITLE
You don't have an extension for debugging Python

### DISCUSSION THREAD

**Discussion by peter812** [coder/code-server#5804] [Q&A]
"You don't have an extension for debugging Python. Should we find a Python extension in the Marketplace?"

I have Python for VSCode v0.2.2 installed, what am I missing

Server:
docker image:  linuxserver/code-server:4.8.3
ubuntu 20.04 in VM
docker from snap store
(No proxy, http only)

Clients:
win 10 computers
Microsoft edge

**Comment by lg-nineoneninethreenine** [comment]
A video or photo mayhaps? We can't really understand the error you are recieving.

Try making a video of the error, I will check soon.

**Reply by jsjoeio** [reply]
Yes! Can you share steps to reproduce and a video?

**Reply by lg-nineoneninethreenine** [reply]
Um @peter812? We need you help as much as you need ours. I have a lot of exams coming up so I can't tell if I can work on my computer, but sharing a video might help us guess the issue.

**Reply by peter812** [reply]
'configuration.jsonValidation.url' must be an absolute URL or start with './' to reference schemas located in the extension.
[https://youtu.be/K7l_MdfRuPY](url)
sorry for the wait

**Reply by jsjoeio** [reply]
Hmm seems like there is something blocking some of the extension request data. Not sure if it's due to insecure context. You could also try going to https://open-vsx.org/ to download the vsix directly and installing that way. I might try that if I were you.
